renderdoc/data/glsl/debuguniforms.h

```diff
@@ -26,9 +26,11 @@
 // classes that represent a whole cbuffer
 #if defined(__cplusplus)
 
-#include "common/common.h"
-#include "maths/matrix.h"
-#include "maths/vec.h"
+// The Android GL ES shader compiler does not supports the " character.
+// The includes should be before this file is included!
```

Comment:
I'm a bit confused by this - does this mean that android doesn't allow " to appear anywhere in the file at all, even in a preprocessor block that it's not compiling?!